### PR TITLE
CU-296cyym - xcvm

### PR DESCRIPTION
**Pull request by hussein-aitlahcen**
Draft PR, only purpose is to be sure CI is always passing.

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-296cyym Demo](https://app.clickup.com/t/296cyym)

File: xcvm/xcvm-instructions/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+#[derive(Copy, Clone)]
+pub enum XCVMCosmChain {
+	Terra,
+}
+
+#[derive(Copy, Clone)]
+pub enum XCVMNetwork {
+	Substrate(XCVMSubstrateChain),
+	Cosmos(XCVMCosmChain),
+}
```
Comment (KaiserKarel) [inline xcvm/xcvm-instructions/src/lib.rs:37]: I would simplify this a bit, perhaps:

```
pub enum Network {
    Picasso,
    Osmosis,
    Custom(u32)
}
```

Where we can doing storage accesses for the well know networks and hardcode routes (Picasso <> KSM), but for future use, add more networks with extrinsics.

Comment (cor) [reply xcvm/xcvm-instructions/src/lib.rs:37]: Resolved by changing to

```
#[repr(transparent)]
pub struct XCVMNetwork(u8);
```

**Comment by cor** [comment]
Replaced by #1033

**Pull request closed without merging**